src/guc.c

```diff
@@ -292,6 +293,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable("timescaledb.enable_tiered_reads",
```

Comment:
The setting is called `enable_tiered_reads` the variable is called `ts_guc_enable_osm_reads`. Is this different terminology (tiered vs. osm) intentional?

Reply:
It's intentianal. We decided that we don't want to expose the name "OSM" in the user's api, but internally we refer to it as OSM.